frontend/src/Constants/cart.js

```diff
@@ -0,0 +1,8 @@
+export const ADD_TO_CART = "ADD_TO_CART";
```

Comment:
Can you tell me why did you do it this way instead of using them directly in the reducer functiuon?